spec/request/user/update_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe 'PATCH #update user params', type: :request do
+  let!(:user) { create(:user, :confirmed) }
```

Comment:
I think you should add the confirmed as default, so you don't need to add the trait every time you use a user.
Generally, the user that you need, should be confirmed, right?
You can create a `trait` for not confirmed ones. WDYT?

Reply:
Great, how about, insted of a `trait`, a before block that works for that context?
The trait might be... tricky. @gonzaM

Reply:
Do you mean a context with a user with the confirmed_at nil? 👍